tests/test_jmespath_expression.py

```diff
@@ -1,7 +1,26 @@
 from datayoga.blocks.expression import JMESPathExpression
 
+expression = JMESPathExpression()
 
-def test_sql_expression():
-    expression = JMESPathExpression()
-    expression.compile("concat([fname,' ',mname,' ',lname])")
-    assert expression.search({"fname": "john",  "mname": "george", "lname": "smith"}) == "john george smith"
+CONCAT_EXPRESSION = "concat([fname,' ',mname,' ',lname])"
+TEST_JSON = {"fname": "JohN",  "mname": "GeorgE", "lname": "SmitH"}
+
+
+def test_jmespath_concat_expression():
+    expression.compile(CONCAT_EXPRESSION)
+    assert expression.search(TEST_JSON) == "JohN GeorgE SmitH"
+
+
+def test_jmespath_capitalize_expression():
+    expression.compile(f"capitalize({CONCAT_EXPRESSION})")
```

Comment:
any reason to mix the concat in the test? Ideally the test should be focused on one thing, so maybe remove the concat from the upper/lower tests?

Reply:
fixed